### PR TITLE
optimize Fatalf with the expectedError

### PR DESCRIPTION
**Pull request by lixiaobing1**
optimize Fatalf with the expectedError.

### CODE REVIEW

File: pkg/httputils/resumablerequestreader_test.go [hunk 1]
```diff
@@ -195,10 +195,11 @@ func TestResumableRequestReaderWithServerDoesntSupportByteRanges(t *testing.T) {
 	}
 	defer resreq.Close()
 
+	expectedError = "the server doesn't support byte ranges"
```
Comment (AkihiroSuda) [inline pkg/httputils/resumablerequestreader_test.go]: This does not compile

**Review by AkihiroSuda — CHANGES_REQUESTED** [review]

**Comment by lixiaobing10051267** [comment]
ok, has been modified.

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Review by AkihiroSuda — APPROVED** [review]
LGTM

**Pull request merged**